expo/expo-abstract.typ

```diff
@@ -0,0 +1,30 @@
+*School of Systems and Enterprises*
+
+*Project Name*
+
+OpenTogetherTube Load Balancer
+
+*Description*
+
+OpenTogetherTube (OTT) is an open source web application that allows users to watch
+videos together in real-time. The nature of the application makes it non-trivial
+to scale to a large number of users. To solve this, the team has designed and
+implemented a custom load balancer to appropriately handle the needs of the project.
+The load balancer allows the application to be deployed around the world, lowering
+latency for users, improving reliability, and allowing for a larger number of
+simultaneous users.
+
+*Members*
+
+- Victor Giraldo
+- Carson McManus
+- Michael Moreno
+- Chris Roddy
```

Comment:
Christopher Roddy
(Just for the purpose of the expo being an official university event)